### PR TITLE
Hubbard model nits

### PR DESCRIPTION
**Pull request by fdmalone**
This was bothering me when mypy complained that $t$ or $u$ (not $\mu$ as $u$ was called as a variable) were floats (which they are in general). Also uses $u$ as in the paper which for some reason doesn't use the capital $U$ hubbard U.

### CODE REVIEW

**Comment by fdmalone** [comment]
Ok I ended up fixing the hubbard model notebook too to use the autogen feature but appended the resource estimation to the end.

File: qualtran/serialization/resolver_dict.py [hunk 1]
```diff
@@ -248,6 +247,7 @@
     "qualtran.bloqs.data_loading.qrom.QROM": qualtran.bloqs.data_loading.qrom.QROM,
     "qualtran.bloqs.data_loading.select_swap_qrom.SelectSwapQROM": qualtran.bloqs.data_loading.select_swap_qrom.SelectSwapQROM,
     "qualtran.bloqs.mod_arithmetic.CModAddK": qualtran.bloqs.mod_arithmetic.CModAddK,
+    "qualtran.bloqs.mod_arithmetic.mod_addition.ModAddK": qualtran.bloqs.mod_arithmetic.mod_addition.ModAddK,
```
Comment (tanujkhattar) [inline qualtran/serialization/resolver_dict.py:250]: Is this intended as part of the PR ?

**Review by tanujkhattar — APPROVED** [review]

File: qualtran/serialization/resolver_dict.py (hunk 1 above)
Comment (fdmalone) [reply qualtran/serialization/resolver_dict.py:250]: PrepareHubbard wouldn't serialized without it, I think the module name was slightly off before.

Comment (fdmalone) [reply qualtran/serialization/resolver_dict.py:250]: In particular, ModAddK moved to mod_arithmetic from arithmetic.

**Comment by mpharrigan** [comment]
fixes #929

**Pull request merged**